src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -129,6 +129,31 @@ private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
             return false;
         }
 
+        private BoundExpression GetFileIndices()
+        {
+            var documents = PooledHashSet<DebugSourceDocument>.GetInstance();
```

Comment:
Never mind, it is a HashSet.

---
In reply to: [115794970](https://github.com/dotnet/roslyn/pull/19365#discussion_r115794970) [](ancestors = 115794970)